core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -143,21 +143,21 @@ class AdminClient(val time: Time,
                              clientHost: String,
                              assignment: List[TopicPartition])
 
-  def describeConsumerGroup(groupId: String): List[ConsumerSummary] = {
+  def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
```

Comment:
For the sake of argument, another option might be to create a `ConsumerGroupSummary` which included assignments and the exact state. This might be overkill at the moment, but I'm curious if we'll have cases in the future where we'd like to have access to the specific state. What do you think?